tools/build_pytorch_libs.sh

```diff
@@ -21,17 +21,17 @@ CMAKE_COMMAND="cmake"
 if [[ -x "$(command -v cmake3)" ]]; then
     if [[ -x "$(command -v cmake)" ]]; then
         # have both cmake and cmake3, compare versions
-        CMAKE_VERSION=$(cmake --version | grep 'cmake version' | awk '{print $NF}')
-        CMAKE3_VERSION=$(cmake3 --version | grep 'cmake3 version' | awk '{print $NF}')
-        CMAKE3_IS_NEWER=$($PYTORCH_PYTHON -c "from distutils.version import StrictVersion; print(1 if StrictVersion(\"${CMAKE3_VERSION}\") >= StrictVersion(\"${CMAKE_VERSION}\") else 0)")
+        CMAKE_VERSION=$(cmake --version | grep 'cmake\d* version' | awk '{print $NF}')
```

Comment:
@pjh5 and I just found out that this won't work because (1) `grep` is missing the regex flag and (2) sometimes cmake prints things after the version number. Could you do either
```
cmake --version | grep -E 'cmake3? version' | awk '{print $3}'
```
or 
```
cmake --version | grep 'version' | awk '{print $3}'
```
instead? Similarly for `cmake3`.

Reply:
Thanks for debugging with @pjh5. To keep it simple I'd go for `cmake --version | grep 'version' | awk '{print $3}'`. Do people agree?

Reply:
Here is a more portable suggestion:
```
cmake --system-information | grep CMAKE_VERSION | awk '{print $2}'
```
This will give you a quoted string containing version number so you would also need to remove the quotes in the python command.

Reply:
I'm fine with whichever choice you decide to do. :)